pandas/core/config_init.py

```diff
@@ -851,7 +857,7 @@ def register_converter_cb(key):
         "format.formatter",
         None,
         styler_formatter,
-        validator=is_instance_factory([type(None), dict, callable, str]),
```

Comment:
umm why doesn't this work?

Reply:
```
x = '{:.2f}'

    def inner(x) -> None:
>       if not isinstance(x, _type):
E       TypeError: isinstance() arg 2 must be a type or tuple of types
```
```
x = <function <lambda> at 0x1251dd160>

    def inner(x) -> None:
>       if not isinstance(x, _type):
E       TypeError: isinstance() arg 2 must be a type or tuple of types
```
Also the signature of of the `is_instance_factory` struggles with callable:
```
>>> isinstance(lambda x: x, callable)
False
>>> isinstance(lambda x: x, type(callable))
False
>>> callable(lambda x: x)
True
```

Reply:
Ok fixed `callable != Callable` !!!